core/db/migrate/20161014221052_add_available_to_columns_and_remove_display_on_from_payment_methods.rb

```diff
@@ -0,0 +1,28 @@
+class AddAvailableToColumnsAndRemoveDisplayOnFromPaymentMethods < ActiveRecord::Migration[5.0]
+  def up
+    add_column(:spree_payment_methods, :available_to_users, :boolean, default: nil)
```

Comment:
What meaning does the default, `nil`, have?

I would prefer that the default to both be `true` (matching the old default behaviour) and that the column was `null: false` to avoid any tri-value logic.